### PR TITLE
[canal13cl] fix info extraction

### PR DESCRIPTION
**Pull request by remitamine**

### CODE REVIEW

File: youtube_dl/extractor/tele13.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+class Tele13IE(InfoExtractor):
+    _VALID_URL = r'^http://(?:www\.)?t13\.cl/videos(?:/[^/]+)+/(?P<id>[\w-]+)'
+    _TESTS = [
+        {
+            'url': 'http://www.t13.cl/videos/actualidad/el-circulo-de-hierro-de-michelle-bachelet-en-su-regreso-a-la-moneda',
+            'md5': '4cb1fa38adcad8fea88487a078831755',
+            'info_dict': {
+                'id': 'el-circulo-de-hierro-de-michelle-bachelet-en-su-regreso-a-la-moneda',
+                'ext': 'mp4',
+                'title': 'El c\u00edrculo de hierro de Michelle Bachelet en su regreso a La Moneda',
```
Comment (jaimeMF) [inline youtube_dl/extractor/tele13.py]: I don't see any reason to use unicode escape sequences here, you can directly use `'í'`.

Comment (remitamine) [inline youtube_dl/extractor/tele13.py]: i copied it from the --write-info-json output.

Comment (jaimeMF) [inline youtube_dl/extractor/tele13.py]: python's `json` use escape sequences, but both are supported. I prefer to directly use the character (imaging having to use escape sequences for titles in chines, arabic ...).

Comment (remitamine) [inline youtube_dl/extractor/tele13.py]: ok i will change it.

**Comment by remitamine** [comment]
i changed the unicode escape sequences and skip the test.

**Pull request merged**